styles.css

```diff
@@ -182,14 +318,26 @@ Cow visiting the Statue of Liberty
 
 	#hero-arrow {
 		font-size: 30px;
-		top: 660px;
+		top: 350px;
 	}
 
 	#about-text p {
 		font-size: 20px;
 
 	}
 
+	/* PORTFOLIO */
+
+	.portfolio .container {
+		width: 100%;
+	}
+
+	.portfolio-item div {
+		width: 550px;
+		height: 300px;
+		border-radius: 0;
+	}
+
 }
 
 @media screen and (max-width: 550px) {
```

Comment:
you need another .portfolio-item tag here; when my browser goes narrower than 550px it goes a bit wonky